src/data/triggers.json

```diff
@@ -131,4 +131,6 @@
   "upmarket",
   "value (add|proposition)",
   "verb(iage|age)"
+  "with upstream"
```

Comment:
hi @ItsPugle, thanks for the PR!

I think you'll need some trailing commas on lines `133` & `134` or the json won't parse

Reply:
Changed - How does that look now? I'm not familiar with json so yeah....